src/applications/mhv-landing-page/components/LandingPage.jsx

```diff
@@ -74,8 +69,16 @@ const LandingPage = ({ data = {}, recordEvent = recordEventFn }) => {
             render={renderMHVDowntime}
           />
           <HeaderLayout showWelcomeMessage={showWelcomeMessage} />
-          {registered && !userHasMhvAccount && <MhvRegistrationAlert />}
-          {registered ? <CardLayout data={cards} /> : noCardsDisplay}
+          <LandingPageAlerts
+            registered={registered}
+            verified={verified}
+            userHasMhvAccount={userHasMhvAccount}
+            userHasMhvBasicAccount={userHasMhvBasicAccount}
+            showsVerifyAndRegisterAlert={showsVerifyAndRegisterAlert}
+            signInService={signInService}
+            cards={cards}
```

Comment:
Per my other comment, I think we'd keep the cards outside of the LandingPage alerts.